src/components/modals/Modal.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import ReactModal from 'react-modal';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
+import ModalHeader from './ModalHeader';
+
+import './Modal.css';
+
+ReactModal.setAppElement('#root');
```

Comment:
I kinda dislike this package looking at this. There's a `React.createPortal` and they didn't use it :(